Adjust src/website/layouts/docs_page.html column classes

lependu opened the pull request:
Hi! Currently in `desktop` view the doc page main column's content overflows the screen width (if the content text is long enough and the screen is not wide enough). 

See the [Typescript and types support](https://www.fastify.io/docs/latest/TypeScript/) page with 770-1400 px wide screen for example. 

The PR changes `bluma` classes to control the max width of the block.

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

lmammino commented:
Hello @lependu. Sorry for the terrible delay in addressing this PR and also thanks for submitting it.

I was testing these changes and I think there's still some "responsive space" that we are not covering properly.

Please let me explain this better with some screenshots:

## width < 768 (OK)

![screen shot 2019-02-13 at 12 09 07 pm](https://user-images.githubusercontent.com/205629/52711030-b88fcb80-2f88-11e9-8fde-bc402b371ad5.png)

## width >= 768 and width < 1024 (still truncating main column)

![screen shot 2019-02-13 at 12 09 23 pm](https://user-images.githubusercontent.com/205629/52711066-cb0a0500-2f88-11e9-90d9-1723f4aa5a4f.png)

## width >= 1024 (OK)

![screen shot 2019-02-13 at 12 09 36 pm](https://user-images.githubusercontent.com/205629/52711116-e543e300-2f88-11e9-81b9-cb057ed61b1f.png)

I think that if we change the classes as follows we might get more consistent results:

 - table of contents column: `column is-one-quarter`
 - main column: `column is-three-quarters`

I am also thinking that it might make sense to move the table of contents at the end of the page for the single column/mobile views.

What do you think?

lependu commented:
Hello @lmammino!

Thank you for the review. I adjusted the classes as you suggested and rebased the branch. It seems OK now on all breakpoints.

I prefer the table of content first approach, but I am afraid it is more about my taste than an established UI design reasoning. In either way we need a lot of scrolling to reach the content or the menu links. 
 
If you prefer to change it, I am happy to create a commit or a new PR.

lmammino commented:
I think we can consider this close by #144 

Feel free to re-open if you see some edge cases still not working properly.

Thanks a lot @lependu :)
